gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilPlugin.kt

```diff
@@ -39,18 +76,18 @@ open class AnvilPlugin : Plugin<Project> {
     // Apply the Anvil plugin after the Kotlin plugin was applied. There could be a timing
     // issue. Also make sure to apply it only once. A module could accidentally apply the JVM and
     // Android Kotlin plugin.
-    project.pluginManager.withPluginOnce("org.jetbrains.kotlin.android") {
-      realApply(project, true)
+    target.pluginManager.withPluginOnce("org.jetbrains.kotlin.android") {
+      realApply(target, true)
     }
-    project.pluginManager.withPluginOnce("org.jetbrains.kotlin.jvm") {
-      realApply(project, false)
+    target.pluginManager.withPluginOnce("org.jetbrains.kotlin.jvm") {
+      realApply(target, false)
     }
 
-    project.afterEvaluate {
+    target.afterEvaluate {
       if (!once.get()) {
         throw GradleException(
             "No supported plugins for Anvil found on project " +
```

Comment:
The message is not very clear (gradle plugin, compiler plugin, how does a plugin requires plugins ?). 
-> The Anvil gradle plugin could not be applied to project...

Reply:
Unrelated to this PR.